tmail-backend/pom.xml

```
@@ -48,6 +48,7 @@
         <module>jmap/extensions-cassandra</module>
         <module>jmap/extensions-es-v7</module>
 
+        <module>rate-limiter</module>
```

```suggestion
        <module>rate-limiter/rate-limiter-api</module>
```

We will get a separate module for Cassandra implem, that's for sure.